src/routes/users/index.js

```diff
@@ -19,7 +20,7 @@ router.get('/collaborators', transactionWrapper(getPossibleCollaborators));
 router.get('/stateCodes', transactionWrapper(getPossibleStateCodes));
 router.get('/statistics', transactionWrapper(getUserStatistics));
 router.get('/active-users', transactionWrapper(getActiveUsers));
-
+router.get('/training-report-users', transactionWrapper(getTrainingReportUsers)); // todo - update redoc with new route
```

Comment:
add e2e-api test for new route

Reply:
https://github.com/HHS/Head-Start-TTADP/pull/1559/commits/1c4e1a6469d7372c4bc9e07bb013684a998a98af